blockchain/blockchain.go

```diff
@@ -236,6 +239,14 @@ func (b *Blockchain) VerifyBlock(block *core.Block) error {
 }
 
 func (b *Blockchain) verifyBlock(txn db.Transaction, block *core.Block) error {
+	if len(block.ProtocolVersion) > 0 {
```

Comment:
what do you think about extracting that logic into a separate function? 
Then it could look like this:

```
	if err := checkBlockVersion(block); err != nil {
		return err
	}
```